### PR TITLE
Issue #66: Add coordinates

### PR DESCRIPTION
**Pull request by meganfung38**
Issue #66 :  
- assigns a location to newly created facilities on facility-form screen 

What to expect:
- map view of user's current location and pin marker (represents location that will be assigned to the newly created facility) 
- drag pin marker to relocate location of facility 
- press pin marker to see its address in a modal 
       - if location of pin marker is a building, address will be name of building 
       - if not, address will be the pin marker 

**NOTE: using mapView callout component did not work out-- cannot figure out how to make address fit inside callout (no style container to fix issue)

### CODE REVIEW

**Review by dylantompkins — CHANGES_REQUESTED** [review]
After initial testing on phone, a few things:
1. It's hard to pinch to zoom on the map because it's small. I think doubling the height would be reasonable.
2. The default zoom of the map is pretty zoomed out. Maybe making the map bigger will zoom it in. If not, default zoom should be increased IMO.
3. The time displays now show seconds. I don't think they did before. Also, the selector doesn't allow you to choose the seconds. It should probably just go back to minutes.
4. The scroll view does not allow you to scroll all the way down.

**Comment by dylantompkins** [comment]
image
- Scrolled all the way down (see scroll bar on the right)
- Time bubbles show seconds

image
- Time selector does not allow seconds, creating a UX discrepancy

**Comment by meganfung38** [comment]
What to expect (after PR requested changes): 
- map view of user's current location and pin marker (represents location that will be assigned to the newly created facility)
- map view should be zoomed enough to see nearby buildings
- map view height should be a reasonable height for map maneuvering
- scroll view should work
- drag pin marker to relocate location of facility
- press pin marker to see its address in a modal
- if location of pin marker is a building, address will be name of building
- if not, address will be the pin marker
- time selector should not show seconds (format: HH: MM)

File: navigation/screens/facility-form.tsx [hunk 1]
```diff
@@ -93,18 +120,75 @@ export default function FacilityForm() {
...
+                    console.error("Error fetching address:", error);
+                  }
+                }}
+              ></Marker>
+            </MapView>
+          ) : (
+            <Text style={styles.subtext}>Fetching current location...</Text>
+          )}
+
+          <Text>press and drag pin marker to see address and relocate</Text>
```
Comment (lindlands) [inline navigation/screens/facility-form.tsx:171]: Nit: capitalize Press

File: navigation/screens/facility-form.tsx [hunk 2]
```diff
@@ -179,6 +257,10 @@ const styles = StyleSheet.create({
     backgroundColor: "#EEF8F7",
     height: "100%",
   },
+  scrollView: {
```
Comment (lindlands) [inline navigation/screens/facility-form.tsx:260]: There is a lot of scroll left at the bottom of the page with no content. Can this be fixed?

**Review by lindlands — CHANGES_REQUESTED** [review]

**Review by dylantompkins — APPROVED** [review]
Looks great!!

**Review by lindlands — APPROVED** [review]
Nice job!!

**Pull request merged**